src/js/webrequest.js

```diff
@@ -566,9 +566,9 @@ function _isTabChromeInternal(tabId) {
  */
 function _isTabAnExtension(tabId) {
   return (
-    _frameUrlStartsWith(tabId, "chrome-extension://") ||
-    _frameUrlStartsWith(tabId, "moz-extension://")
-   );
+   _frameUrlStartsWith(tabId, "chrome-extension://") ||
+   _frameUrlStartsWith(tabId, "moz-extension://")
+  );
```

Comment:
This is a big-time nit, buuuut, removing the space from the closing parenthesis is great, that was a typo, but creating single-space indent above isn't so great; an indentation level should be two spaces.